docs/manual-EN/server-administration/storage-balance.md

```diff
@@ -0,0 +1,219 @@
+| 192.168.8.210 | 34600 | online | 0            | test: 0             | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34700 | online | 52           | test: 52            | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34500 | online | 48           | test: 48            | test: 100              |
+------------------------------------------------------------------------------------------------
+Got 3 rows (Time spent: 6029/7175 us)
+```
+
+### Step 2 Start new storage instances
+Start 5 new storaged instances to scale out, show the new status using command `SHOW HOSTS`.
+
+```
+(user@127.0.0.1) [(none)]> SHOW HOSTS
+
+================================================================================================
+| Ip            | Port  | Status | Leader count | Leader distribution | Partition distribution |
+================================================================================================
+| 192.168.8.210 | 34600 | online | 0            | test: 0             | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34900 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 35940 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34920 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 44920 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34700 | online | 52           | test: 52            | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34500 | online | 48           | test: 48            | test: 100              |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34800 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+```
+
+No partition is distributed on the newly started storage instances.
+### Step 3 Data migration
+List the present balance plan's id using command `balance data`, a new plan id will be generated if new intances are added, no operations will be conducted for balanced clusters.
```

Comment:
You can check the current balance plan id by command `balance data` if the cluster is balanced. Otherwise, a new plan id will be generated by the command.